app/controllers/RegularContributions.scala

```diff
@@ -54,13 +56,19 @@ class RegularContributions(
             )
           )
       }
-    } getOrElse InternalServerError
+    } fold (
+      { error =>
+        SafeLogger.error(scrub"Failed to display recurring contributions form for ${request.user.id} due to error from identityService: $error")
+        InternalServerError
```

Comment:
Perhaps we can do something better than throwing a 500 when we fail to retrieve user details from identity, but it will be useful to know how often this is happening first.